site/pages/badger-profile/index.js

```diff
@@ -27,7 +27,7 @@ const BadgerProfile = ({ badger }: { badger: Badger }) => {
       <div className={styles.profileContainer}>
         <div className={styles.profilePictureContainer}>
           <img
-            src={badger.imageUrl}
+            src={badger.secondaryImageUrl || badger.primaryImageUrl}
```

Comment:
Well no I mean - will we potentially end up in a situation when image is not made / available, but the page must be displayed? Do we need a placeholder then?